logbook-openfeign/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.zalando</groupId>
+        <artifactId>logbook-parent</artifactId>
+        <version>2.14.0-SNAPSHOT</version>
```

Comment:
@sanyarnd you missing relative path here:

```
<relativePath>../logbook-parent/pom.xml</relativePath>
```

Reply:
There's no relative path in any other pom.xml, what's so special about this new one?

Reply:
structure of the maven project was changed, please pull changes from the main branch and adopt your new module to them.

Reply:
Yeah, I see it now. Rebased.